### DISCUSSION TITLE
Wrongly render / build of the page

### DISCUSSION THREAD

**Discussion by polykelvin** [vuepress/core#1375] [Q&A] [answered]
Hello, I am trying to build my website using `pnpm docs:build`; however, the result is rendering incorrectly. The website is not formatted correctly.
![image](https://github.com/vuepress/vuepress-next/assets/36104776/0dc065d4-e37d-4da0-92fb-e700cd81f4d4)

- Ensured running under correct environmentm. node, pnpm, version requirement satisfied 
- updated using `pnpm dlx vp-update`

Please let me know if there is anything I should be aware of.

**Comment by Mister-Hope** [comment]
base

**Reply by polykelvin** [reply]
Thank you for replying, I had tried different base setting including the default one ("/") and the the one with repo ("/REPO/"), both of them have the same issue. I am also aware that after I build the website locally(run the deploy.sh), the index.html in the "dist" folder have the same rendering issue.

**Reply by meteorlxy** [reply]
Did you open the `dist/index.html` file directly?

**Reply by polykelvin** [reply]
Yes, I have opened the `dist/index.html` file directly. I have also tried accessing `username.github.io/repo`, but both methods result in the same rendering problem.

**Comment by polykelvin** [comment, marked as answer]
My solution is to move all files from the default `my-docs` to the root folder of your repo.
